STYLE_GUIDE.md

```diff
@@ -72,33 +72,39 @@ do_something(foo: 1,
   The parameters on a multi-line method call or definition are aligning by an indentation level.
   <sup>[[link](#style-parameters-aligning)]</sup>
 
+  **Recommended:** write each parameter on the separate line.
+
 ```ruby
-# good
+# bad
 
-def foo(bar,
-  baz)
+def foo(bar, baz,
+        kek, lol)
   123
 end
 
+# bad
+
 def foo(
   bar,
-  baz
-)
+     baz)
   123
 end
 
-# bad
+# good
 
-def foo(bar,
-        baz)
+def foo(bar , baz,
```

Comment:
@roman-dubrovsky there's a redundant space before comma I think

Reply:
yep, this is my perfect keyboard